content/en/docs/setup/best-practices/certificates.md

```diff
@@ -21,23 +21,28 @@ This page explains the certificates that your cluster requires.
 
 Kubernetes requires PKI for the following operations:
 
-* Client certificates for the kubelet to authenticate to the API server
-* Kubelet [server certificates](/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/#client-and-serving-certificates)
-  for the API server to talk to the kubelets
+### Server Certificates
 * Server certificate for the API server endpoint
-* Client certificates for administrators of the cluster to authenticate to the API server
-* Client certificates for the API server to talk to the kubelets
-* Client certificate for the API server to talk to etcd
-* Client certificate/kubeconfig for the controller manager to talk to the API server
-* Client certificate/kubeconfig for the scheduler to talk to the API server.
-* Client and server certificates for the [front-proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/)
+* Server certificate for the ETCD server
+* [Server certificates](/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/#client-and-serving-certificates) for the Kubelets (one for each worker node) so the API server can communicate with the Worker Nodes
+* Server certificate for the [front-proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/)
+
+### Client Certificates
+
+* Client certificates for the Kubelet to authenticate to the API server
+* Client certificate for the API server to securely communicate with the ETCD server
+* Client certificate for the Controller Manager to securely communicate with the API server
+* Client certificate for the Scheduler to securely communicate with the API server
+* Client certificates for Administrators of the Cluster to authenticate to the API server
+* Client certificates for kube-proxy to authenticate to the API server
+* Client certificate for the [front-proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/)
 
```

Comment:
Does the API server need a certificate to authenticate itself to the kubelets?

Reply:
To establish a secure connection and authenticate itself to the kubelet, the kube-apiserver requires a client certificate and key pair. In this scenario, there are two approaches for certificate usage: using shared certificates or separate certificates; 

- Shared Certificates: The kube-apiserver can utilize the same certificate and key pair it uses to authenticate its clients. This means that the existing certificates, such as `apiserver.crt` and `apiserver.key`, can be used for communicating with the kubelet servers.

- Separate Certificates: Alternatively, the kube-apiserver can generate a new client certificate and key pair to authenticate its communication with the kubelet servers. In this case, a distinct certificate named `kubelet-client.crt` and its corresponding private key, `kubelet-client.key` are created.

Reply:
OK, I recommend mentioning that in the docs.
In other words: tell the readers, not just the reviewers.

Reply:
@sftim I've added the explanation as suggested in my latest commit. Let me know if it looks good.